app/src/components/Schedule.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div class="wrapper" v-if="normal">
+        <div id="current" class="text-center current schedule">
+            <h3>Τώρα <i class="fas fa-users-class"></i></h3>
```

Comment:
We should look into localization to support multiple languages for the front part at least.
The library https://kazupon.github.io/vue-i18n/ seems to be able to do the trick.

Reply:
@root-expert check it out.

Reply:
I dit it. @enderian ΦΤΙΑΞΕ ΤΑ ΑΓΓΛΙΚΑ!

Reply:
Ναι είναι καλά ο αδελφός μου για την μετάφραση στα αγγλικά xD